scripts/grafana-server/start-server

```diff
@@ -23,8 +23,9 @@ fi
 echo starting server
 
 cp -r ./bin $RUNDIR
-cp -r ./public $RUNDIR
 cp -r ./tools $RUNDIR
+ln -s $(realpath ./public) scripts/grafana-server/tmp
```

Comment:
Are `$RUNDIR` and `scripts/grafana-server/tmp` the same?

Reply:
Looks like it (since this worked in ci), but we should use the variable instead. I'll fix.